tests/test_xml.py

```diff
@@ -0,0 +1,117 @@
+import os
+from xml.etree import ElementTree
+
+from maltego_trx.maltego import MaltegoTransform
+from maltego_trx.overlays import OverlayType, OverlayPosition
+from maltego_trx.server import build_exception_message
+
+XML_SAMPLES_DIR = os.path.join(os.path.dirname(__file__), 'xml_samples')
+
+
+def test_entity_with_value():
```

Comment:
For snapshot testing, you can think about using a pytest library to remove some of the boilerplate.

https://tophat.github.io/syrupy/

Would make the tests shorter, as you wouldn't need to repeat the file reading and comparison logic for every test.

Reply:
Very nice, only knew the snapshottests lib. Will use this